docs/reference/api/docker_remote_api.md

```diff
@@ -116,6 +116,8 @@ This section lists each version from latest to oldest.  Each listing includes a
 
 [Docker Remote API v1.25](docker_remote_api_v1.25.md) documentation
 
+* `POST /containers/create` now takes `AutoRemove` in HostConfig, auto-removal will be done on daemon side.
```

Comment:
Maybe we should remove "or the client disconnects"? I think it's not very accurate, detach a container or unexpected client crash won't make the container removed. 

And do you think we should mention the auto-removal will happen on daemon side? Because this is the main change of this PR.